compiler/z/codegen/OMRInstruction.cpp

```diff
@@ -85,7 +85,7 @@ OMR::Z::Instruction::Instruction(TR::CodeGenerator* cg, TR::InstOpCode::Mnemonic
    OMR::Instruction(cg, op, node),
    CTOR_INITIALIZER_LIST
    {
-   TR_ASSERT(cg->getS390ProcessorInfo()->supportsArch(_opcode.getMinimumALS()), "Processor detected does not support instruction %s\n", _opcode.getMnemonicName());
+   TR_ASSERT_FATAL(cg->getS390ProcessorInfo()->supportsArch(_opcode.getMinimumALS()), "Processor detected does not support instruction %s\n", _opcode.getMnemonicName());
```

Comment:
Can we as part of this change also modify `LGG` and `LLGFSG` to be marked as "unknown architecture" instructions in OMRInstOpCodeProperties.hpp? This will avoid OpenJ9 issues where the GC and the JIT are not synced up properly w.r.t. z14 support.

Reply:
Sure, done in 9b98afc